src/main/resources/application-dev.yml

```
@@ -0,0 +1,22 @@
+spring:
+  application:
+    name: Disaster Alerts
+
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: jdbc:postgresql://db:5432/disasterdb
```

```suggestion
    url: jdbc:postgresql://localhost:5432/disasterdb
```
sprawdź czy rozwiąże to problem z tym testem